app/app.js

```diff
@@ -0,0 +1,19 @@
+const { getTopics } = require("../controllers/topics-controllers");
+
+const express = require("express");
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.status && err.mssg) {
+    res.status(err.status).send({ msg: err.msg });
+  }
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Bad request" });
+  } else {
+    next();
+  }
+});
```

Comment:
Nice that you are thinking of future error handling, but please only put these in when the time comes. Otherwise, you will auto pass your error handling tests! Please do delete the error handlers for now